### DISCUSSION TITLE
skyffel - prototype for generating Airbyte connectors

### DISCUSSION THREAD

**Discussion by philipandersson** [airbytehq/airbyte#35876] [Show and tell]
My co-founder and I at [skyffel](https://skyffel.com/) have been playing around with building AI agents in the data pipelining space over the last week and just got a first working prototype! We help you tackle data loading from HTTP API.

The agent generates an Airbyte low-code (YAML) connector from documentation. All you have to do is to provide it with the goal of your integration and links to documentation.

Take it for a spin and let us know what you think: https://github.com/skyffel/airbyte-connector-generator-poc

https://github.com/airbytehq/airbyte/assets/3134895/f4ab4833-bdad-48e9-9b62-098a96555d46

_Disclaimer - it is just a Proof of Concept for now_

**Comment by marcosmarxm** [comment]
Interesting, but the example uses an API with only one endpoint that doesn't consume data. I'd love to see a more complex example to showcase the tool's full potential. Nice proof of concept! 🚀

**Reply by philipandersson** [reply]
Absolutely! Any ideas are welcomed, we want to support the Airbyte community and make it production-grade. Feel free to add an issue.

We've started collecting some inital feedback here: [#improvement](https://github.com/skyffel/airbyte-connector-generator-poc/issues/2)
